packages/test/package.json

```diff
@@ -6,6 +6,7 @@
   "scripts": {
     "build": "tsc --build",
     "build.watch": "tsc --build --watch",
+    "build.proto": "pbjs -t static-module -w commonjs -o protos/protobufs.js protos/test-messages.proto && pbts protos/protobufs.js -o protos/protobufs.d.ts",
```

Comment:
I'd have this as part of `build` that's the step where we compile the core and api protos.

Reply:
And let's not commit the generated files please.

Reply:
Also, will this work on windows?